auctioneerrpc/auctioneer.proto

```diff
@@ -887,10 +887,18 @@ message ServerBid {
     uint32 version = 4;
 
     /*
-    The minium node tier this order should be matched with. Only asks backed by
+    The minimum node tier this order should be matched with. Only asks backed by
     a node this tier or higher will be eligible for matching with this bid.
     */
     NodeTier min_node_tier = 5;
+
+    /*
+    If this bid order is meant to lease a channel for another node (which is
+    dubbed a "sidecar channel") then this ticket contains all information
+    required for setting up that sidecar channel. The ticket is expected to be
+    the raw bytes of the ticket tlv stream without any prefix or checksum.
+    */
+    bytes sidecar_ticket = 6;
```

Comment:
See my comment re also allowing some of this information to be specified w/o the raw encoding of the ticket. Alternatively, we can make a help er function to construct the ticket, but with the same information of the node creating it as both the offerer and the acceptor.

Reply:
Also it could be that we accept the sidecar ticket on the level of the client protos, but instead send out the individual fields between the client and the actual server.

Reply:
I've done this in the latest iteration :+1: